<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>decode</class>
 <widget class="QWidget" name="decode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_cip">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>751</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>密文</string>
   </property>
   <widget class="QPlainTextEdit" name="cipherText">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>20</y>
      <width>731</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_msg">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>751</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>明文</string>
   </property>
   <widget class="QPlainTextEdit" name="messageText">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>20</y>
      <width>731</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_param">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>220</y>
     <width>661</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>参数</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>19</y>
      <width>641</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QPlainTextEdit" name="text_N">
       <property name="plainText">
        <string>012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_N">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>N</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_d">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>d</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPlainTextEdit" name="text_d">
       <property name="plainText">
        <string>012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_decode">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>71</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>解
密</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
